definitions/npm/@reduxjs/toolkit_v1.x.x/flow_v0.104.x-/toolkit_v1.x.x.js

```diff
@@ -0,0 +1,194 @@
+declare module '@reduxjs/toolkit' {
+  import type { AnyAction, Action, Reducer } from 'redux';
```

Comment:
Sadly that can't work, those types will all resolve to `any`. 
See https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#dont-import-types-from-other-libdefs 

**suggestion:** copy paste those types here in the lifdef...

Reply:
Maybe using [this workaround](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#dont-import-types-from-other-libdefs), so we have some sort of "convention".

Reply:
Inlined them now.